Source/Core/Core/PowerPC/JitArm32/JitArm_LoadStore.cpp

```diff
@@ -464,7 +464,7 @@ void JitArm::lXX(UGeckoInstruction inst)
 		rA = gpr.GetReg();
 
 		MOVI2R(rA, (u32)&PowerPC::OnIdle);
-		MOVI2R(R0, PowerPC::ppcState.gpr[a] + (s32)(s16)inst.SIMM_16);
+		MOVI2R(R0, PowerPC::ppcState.gpr[a] + (s32)(s16)inst.SIMM_16); // Temp Fix
```

Comment:
All this does is append `// Temp Fix` to the end of the line. Nothing else is changed.